libr/core/cconfig.c

```diff
@@ -566,7 +566,13 @@ static int cb_asmbits(void *user, void *data) {
 	}
 
 	bits = node->i_value;
-
+#if 0
```

Comment:
what about removing this? code is probably not that useful since it breaks stuff... if it worked it wouldn't have been commented out

Reply:
it breaks stuff for a reason that must be investigated i prefer to keep the comment there for future review